integration

```diff
@@ -641,7 +642,6 @@
   "mletenay/home-assistant-ev-charge-control",
   "mletenay/home-assistant-goodwe-inverter",
   "mmillmor/geo_home",
-  "modrzew/hass-flashforge-adventurer-3",
```

Comment:
This is not a transfer, it is a separate repository.
As such, do not remove and add a repository in the same PR.

Reply:
Moved the removal to #2730

Reply:
Hey @ludeeus can you take a look again, and at #2730 as well? I'm not really sure what the process is for getting this reviewed and merged since the PR description said _DO NOT REQUEST REVIEWS, THAT IS JUST RUDE, IF YOU DO THE PULL REQUEST WILL BE CLOSED!_ 😅